packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -163,7 +163,10 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         require(_getIdentifierWhitelist().isIdentifierSupported(identifier), "Unsupported identifier");
         require(_getCollateralWhitelist().isOnWhitelist(address(currency)), "Unsupported currency");
         require(timestamp <= getCurrentTime(), "Timestamp in future");
-        require(stampAncillaryData(ancillaryData, msg.sender).length <= ancillaryBytesLimit, "Ancillary Data too long");
+        require(
+            _stampAncillaryData(ancillaryData, msg.sender).length <= ancillaryBytesLimit,
```

Comment:
- The `requestPrice` function of the `OptimisticOracle` now [calls `stampAncillaryData`](https://github.com/UMAprotocol/protocol/blob/43d569f66af0323813716927ef2dcc8c7f60dd33/packages/core/contracts/oracle/implementation/OptimisticOracle.sol#L166), which is a pass-through function for the internal [`_stampAncillaryData` function](https://github.com/UMAprotocol/protocol/blob/43d569f66af0323813716927ef2dcc8c7f60dd33/packages/core/contracts/oracle/implementation/OptimisticOracle.sol#L669). Consider calling the internal function directly for simplicity and consistency with the rest of the contract.